sources/Debate/src/main/java/com/example/Debate/dto/NotificationDto.java

```diff
@@ -0,0 +1,23 @@
+package com.example.Debate.dto;
+
+import com.example.Debate.model.Activity;
+import com.example.Debate.model.Punishment;
+import com.example.Debate.model.User;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.data.mongodb.core.mapping.DBRef;
+
+
+@Getter
+@Setter
```

Comment:
same as above - the dto's, in my opinion, should be immutable. a @Value annotation should be used here